packages/app/src/messages/components/thread-details-messages.component.tsx

```diff
@@ -85,11 +86,27 @@ const ThreadDetailsMessages: FunctionComponent<Properties> = ({
   let observer: IntersectionObserver
 
   const callback = (entries: IntersectionObserverEntry[]) => {
+    // notification when user during scroll
+    setOnBottom(entries[0].isIntersecting)
     if (entries[0] && entries[0].isIntersecting) {
       closeNewMessageBadge()
     }
   }
 
+  useEffect(() => {
```

Comment:
Shouldn't we add some tests for that (the `if` conditions seems to be quite long so that maybe it would be good to ensure it works fine)?

Reply:
It's hard to test this component, let's do it as part of the tech task to not block this pr. What do you think? 🤔

Reply:
Sure, no problem. Maybe it should be tested as a part of e2e test. Not a super prio - I would even consider not to add a separate task for that.